app/internal/page/certificate_provider_address.go

```diff
@@ -117,36 +117,28 @@ func readCertificateProviderAddressForm(r *http.Request) *certificateProviderAdd
 	return d
 }
 
-func (d *certificateProviderAddressForm) Validate() validation.List {
+func (f *certificateProviderAddressForm) Validate() validation.List {
```

Comment:
I'm 99% sure all the address forms are the same, so want to consolidate them (in a separate PR)